app/components/ReportsTable/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useCallback } from 'react';
+import { AgGridReact } from '@ag-grid-community/react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { ColumnState } from '@ag-grid-community/core';
+
+import useGrid from '@/hooks/useGrid';
+import { ReportType } from '@/context/ReportContext';
+import LaunchCell from '@/components/LaunchCell';
+import NoRowsOverlay from '@/components/DataTable/components/NoRowsOverlay';
+import columnDefs from './columnDefs';
+
+import './index.scss';
+
+/**
+ * Report table containing all reports
+ */
+const ReportsTableComponent = (reportsData: ReportType[]): JSX.Element => {
```

Comment:
more of a chore, but shouldn't we destructure props into named ones?

Reply:
Hmm when I do that I got this error in React: Cannot destructure property 'rowData' of '_ref' as it is undefined